### DISCUSSION TITLE
Looking for instructions to Modify Display.

### DISCUSSION THREAD

**Discussion by broderp1** [mriscoc/Ender3V2S1#767] [General]
Before you link me to the page showing how to update the display here:
https://github.com/mriscoc/Ender3V2S1/tree/Ender3V2S1-Released/display%20assets

This is not what I am asking.  :)   I want to customize and make my own icons for my printer (and share them of course).

I see the file structure and see the files but am looking for information about what is needed to create the required files, what the files are for and how to compile and create my own DWIN set.  

Any links to what is needed to customize icons and the text?  (Specific size requirements, compiling instructions etc.) It can't be as simple as opening the .ICO file and editing it. Can it?

**Comment by mriscoc** [comment]
https://github.com/b-pub/dwin-ico-tools
